content/webapp/components/Exhibition/Exhibition.tsx

```diff
@@ -287,7 +289,21 @@ const Exhibition: FunctionComponent<Props> = ({ exhibition, pages }) => {
       {exhibition.end && !isPast(exhibition.end) && (
         <InfoBox title="Visit us" items={getInfoItems(exhibition)}>
           <AccessibilityServices>
-            <a href="/access">All our accessibility services</a>
+            For more information, please visit our{' '}
+            <a href="/access">Accessibility</a> page. If you have any queries
```

Comment:
Not especially important if there are unexpected complications - perhaps there's a reason it's not like this already - but I wonder if we use this opportunity to try using a [`next/link`](https://nextjs.org/docs/api-reference/next/link) here (and above) rather than a plain `a` tag? There are plenty of examples around in the repo for prior art.

Reply:
Wondered the same but didn't explore - will have a play around now.